hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import styles from "./CartCard.module.scss";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+import CardMedia from "@material-ui/core/CardMedia";
+import CardContent from "@material-ui/core/CardContent";
+import CardActions from "@material-ui/core/CardActions";
+import Select from "@material-ui/core/Select";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+
+const Selections = (currentStock) => {
+    const items = [];
+    let i = 0;
+
+    for (i = 1; i < currentStock + 1; i++) {
+        items.push(
+            <MenuItem key={i} role="quantity" value={i}>
+                {i}
+            </MenuItem>
+        );
+    }
+    return items;
+};
+
+const Stock = ({ currentStock, handleChange, item }) => {
+    if (!currentStock) {
+        return (
+            <Typography variant="caption" className={styles.CartError}>
+                Currently unavailable
+            </Typography>
+        );
+    } else {
+        return (
+            <FormControl
+                variant="outlined"
+                size="small"
+                className={styles.CartFormControl}
+            >
+                <InputLabel shrink={true} id="Qty">
+                    Qty
+                </InputLabel>
+                <Select
+                    label="Qty"
+                    value={item}
+                    onChange={handleChange}
+                    role="selecter"
+                    name="quantity"
+                >
+                    {Selections(currentStock)}
+                </Select>
+            </FormControl>
+        );
+    }
+};
+
+export const CartCard = ({
+    image,
+    title,
+    currentStock,
+    closeChange,
+    checkedOutQuantity,
+    isError,
+}) => {
+    const [item, setItem] = React.useState(checkedOutQuantity);
```

Comment:
This is fine for now, but eventually this will have to be replaced with a Formik wrapper that handles the form state. Do you want to do that in a separate PR? You can take a look at what that would involve in the `ProductOverview` component: https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.js#L80

Reply:
Lets do this in a separate PR